pkg/products/grafana/reconciler.go

```diff
@@ -417,3 +418,21 @@ func (r *Reconciler) reconcileHost(ctx context.Context, serverClient k8sclient.C
 
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func GetGrafanaConsoleURL(ctx context.Context, serverClient k8sclient.Client, installation *integreatlyv1alpha1.RHMI) (string, error) {
+
+	grafanaConsoleURL := installation.Status.Stages[integreatlyv1alpha1.ProductsStage].Products[integreatlyv1alpha1.ProductGrafana].Host
+	if grafanaConsoleURL != "" {
+		return grafanaConsoleURL, nil
+	}
+
+	ns := installation.Spec.NamespacePrefix + defaultInstallationNamespace
```

Comment:
I know that you are using the one in the file here which was already there but we have these centralised here: https://github.com/integr8ly/integreatly-operator/blob/c7c20a4d49150d304257816756787040c593f4a3/test/common/types.go#L45 I think we should try to use that instead.

Reply:
I agree, although I thought these variables were just used for the automated tests. If we are going to use them I'd suggest we  move it somewhere that is not under the test package, wdyt?

Reply:
I think we'll be addressing namespace stuff in the next sprint/release so for me this is fine as we are using this as you have it here.